plugins/tokenizers/mecab.c

```diff
@@ -520,7 +520,8 @@ mecab_model_create(grn_ctx *ctx, grn_mecab_tokenizer_options *options)
 
   argv[argc++] = "Groonga";
   if (!need_default_output) {
-    argv[argc++] = "-Owakati";
+    argv[argc++] = "-F%m\t";
```

Comment:
Can we register a word that includes a `\t` to MeCab?

Reply:
We could register it. Should we use the other word as a delimiter?
```
$ cat word.csv
search engine,1358,1358,-1,名詞,一般,*,*,*,*,ペン,ペン,ペン,,
search	engine,1358,1358,-1,名詞,一般,*,*,*,*,ペン,ペン,ペン,,
$ mecab
search engine
search engine	名詞,一般,*,*,*,*,ペン,ペン,ペン,,
EOS
search	engine
search	engine	名詞,一般,*,*,*,*,ペン,ペン,ペン,,
EOS
```

Reply:
I will recheck the word we cannot register in MeCab's dictironary.

Reply:
The same problem should occur with full-width spaces in https://github.com/groonga/groonga/pull/2331#discussion_r2088087587 .

Reply:
How about `\n`?

Reply:
Thanks it sounds nice! We could use `\n` in CSV file but we cannot use it as a registered word as follows.
```console
$ cat word.csv
search engine,1358,1358,-1,名詞,一般,*,*,*,*,ペン,ペン,ペン,,
search	engine,1358,1358,-1,名詞,一般,*,*,*,*,ペン,ペン,ペン,,
"search\nengine",1358,1358,-1,名詞,一般,*,*,*,*,ペン,ペン,ペン,,
$ mecab
search\nengine
search\nengine	名詞,一般,*,*,*,*,ペン,ペン,ペン,,
```
```console
$ cat word.csv
search engine,1358,1358,-1,名詞,一般,*,*,*,*,ペン,ペン,ペン,,
search	engine,1358,1358,-1,名詞,一般,*,*,*,*,ペン,ペン,ペン,,
"search
engine",1358,1358,-1,名詞,一般,*,*,*,*,ペン,ペン,ペン,,
$ nkf -e word.csv > word.euc.csv
$ /usr/lib/mecab/mecab-dict-index -d/usr/share/mecab/dic/naist-jdic-eucjp -u user-custom.dic -f euc-jp -t utf-8 word.euc.csv
reading word.euc.csv ... dictionary.cpp(316) [n == 5] format error: "search
```